### ISSUE TITLE
Estimation failed to export diagnostics

### ISSUE THREAD

**Issue by wivern** [OHDSI/Atlas#1990]
Error message is a:
```
Exporting diagnostics
- covariate_balance table

  |                                                                            
  |                                                                      |   0%
  |                                                                            
  |======================================================================| 100%
- preference_score_dist table

  |                                                                            
  |                                                                      |   0%
  |                                                                            
  |======================================================================| 100%
Error in `colnames<-`(`*tmp*`, value = character(0)) : 
  attempt to set 'colnames' on an object with less than two dimensions
Calls: tryCatch ... execute -> exportResults -> exportDiagnostics -> colnames<-
```

**Comment by wivern** [comment]
Analysis design:
[estimation.json.zip](https://github.com/OHDSI/Atlas/files/3447599/estimation.json.zip)

**Comment by wivern** [comment]
Closed by OHDSI/SkeletonComparativeEffectStudy/pull/23
